extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -27,10 +27,22 @@ package stackdriver.config.v1alpha1;
 import "google/protobuf/duration.proto";
 import "google/protobuf/wrappers.proto";
 
-// next id: 10
+// next id: 12
 message PluginConfig {
+  // Types of Access logs to export.
+  enum AccessLogging {
+    // No Logs.
+    None = 0;
+    // All logs only including both success and error logs.
+    FULL = 1;
+    // All error logs when no metadata is available of the peer. This is
+    // currently only available for outbound/client side logs.
+    ERR_ONLY_ON_NO_MX = 2;
```

Comment:
If offloading to log sample filter is the agreed approach, do we need to make this an enum? Shall we keep it as boolean (or a BoolValue if `disable` field name is undesired) and rename it to something without `server` in it, like `enable_access_log`?

Reply:
Offloading to sample filter would involve change in sampling filter and this option was added to fasttrack client side logs.. so that we don't enable client side and server logs by default causing increase in logs cost.

I totally agree with offloading the decision to log to sampling filter rather than stackdriver filter. In case of http,  we can do on presence of header and for tcp on presence of filter state..

Reply:
Judging whether to log based on presence of mx data seems to be the ultimate way of reducing log volume and not sure if that is really necessary for initial release. I think only logging errors (which log sample filter already supports) already seems to be a good budget choice. IMHO we should start with this simple option and avoid complexity. @mandarjog @kyessenov @douglas-reid wdyt?

Reply:
so:  { `NONE`, `ALL`, `ERR_ONLY`} ?  Seems fine to me...

Reply:
Updated based on offline conversation. Changed to Err_Only to log on errors.